File: poc-challenges/250726/src/test/java/com/tdd/domain/FibonacciTest.java
package com.tdd.domain;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class FibonacciTest {
	@Test
	void fibonacci() {
		assertEquals(0, Fibonacci.fib(0));
		assertEquals(1, Fibonacci.fib(1));
		assertEquals(1, Fibonacci.fib(2));
	}
}

File: poc-challenges/260102/src/test/kotlin/com/currenjin/tdd/FibonacciTest.kt
package com.currenjin.tdd

import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class FibonacciTest {
    @Test
    fun fibonacci() {
        assertEquals(1, Fibonacci.fib(1))
        assertEquals(1, Fibonacci.fib(2))
        assertEquals(2, Fibonacci.fib(3))
    }
}